### ISSUE TITLE
Retargeting Ads Designs 

### ISSUE THREAD

**Issue by Nur-alfayez** [EmanHylooz/Abjjad-Product-Marketing#44]
@EmanHylooz - book titles to do designs for ads. (just putting here as a reminder)

**Comment by EmanHylooz** [comment]
I tried to choose from different genres, and added the genre in case there
is capacity to be added somewhere.
I added something small about the book, in case also there is capacity for
that. ​​

   1. الخبز الحافي (بيوغرافيا ومذكرات) هذا العرض لسيرة محمد شكري الذاتية نص
   مؤثر وعمل يحتل موقعاً متميزاً في الأدب العربي المعاصر. وليس صدفة أنه نشر
   بعدة لغات أوروبية كثيرة مثل الإنكليزية أو الإسبانية, قبل نشره بلغته الأصلية
   العربية
   2. رجال في الشمس (أدب فلسطيني  روايات) "لماذا لم يدقوا جدران الخزان."
   3. موتٌ صغير (أدب روايات) ي عام 2017،فازت رواية "موت صغير" للروائي محمد
   حسن علوان بالجائزة العالمية للرواية العربية "البوكر" في دورتها العاشرة.
   4. كيفما فكرت فكر العكس(علم النفس تنمية بشرية وتطوير الذات)  كيفما فكرت
   .. فكر بالعكس يغزو رؤيتنا للعمل والعالم حولنا. بدلاً من النصيحة المملة
   المعتادة، يقدّم بول آردن سخرية جريئة، أقوالاً مأثورة، ومفارقات. يصوّبها على
   "فطرتنا السليمة" ليحثّنا على مراجعتها.
   5. المعجزة أو سبات العقل في الإسلام (قضايا إسلامية) جورج طرابيشي مفكّر
   وكاتب وناقد. يتميّز بكثرة ترجماته ومؤلّفاته حيث ترجم لفرويد وهيجل وسارتر
   وجارودي وسيمون دي بفوار. له مؤلّفات هامّة في الماركسية، وفي النظرية
   القومية، وفي النقد الأدبي للرواية والقصّة العربية، إضافة إلى "معجم
   الفلاسفة"، ومشروعه الموسوعي الضخم الذي عمل عليه أكثر من 15 عاماً وصدرت منه
   أربعة مجلّدات في "نقد نقد العقل العربي".
   6. هكذا تكلم زرادشت (فلسفة تأملات) لخَّص «نيتشه» أفكاره الفلسفية في
   كتابه: «هكذا تكلم زرادشت» الذي قال عنه إنه «دهليز فلسفته». ويعتبر هذا
   الكتاب بحقٍّ علامة من علامات الفلسفة الألمانية
   7. 52 قاعدة عمليَّة للنَّجاح في عملك دونَ أن تخسرَ نفسَك (إقتصاد وأعمال
   إدارة أعمال) هو دليلٌ للأفراد في كلِّ مجالات الحياة لجميع الذين يرغبون في
   إضفاء معنًى على هذه الأوقات المشوِّشة، والتي تتطلَّبُ الاهتمامَ الدقيق
   وتَضَعُ أمامنا تحدِّياتٍ كبرى. باعتماده على خبراته الخاصَّة بصفته محرِّرًا
   مشاركًا في تأسيس مجلَّة ‘‘فاست كومپاني’’ (Fast Company)، وعلى مجالاتِ
   تفاعلاتِه الواسعة مع بعض أعظم المنجِزين والمفكِّرين في العالَم، بِمَن فيهم
   أشخاصٌ حازوا جائزةَ نوبل وأحدثوا تغييراتٍ عالميَّة، أصدرَ ويبر كتابَ ‘‘52
   قاعدة عمليَّة’’- وهو مجموعةٌ تتضمَّن اثنتَين وخمسين قاعدةً تتَّصفُ بالحكمة
   بقَدْر ما هي نافعة، بالصِّدق بقَدْر ما هي مفيدة. إنَّ مصدرَ هذه القواعد هو
   دروسٌ تعلَّمَها ويبر وسجَّلها على بطاقات بحجم 3×5 إنشات (7,5×12,5سم)، وهي
   وسيلةٌ استَعارَها من معلِّميه المرشدين الكثيرين الذين يَصِفُ ويبر تعاليمَهم
   ويُدرجُها في هذا الكتاب.

**Comment by Nur-alfayez** [comment]
Great thanks. 👍

**Comment by Nur-alfayez** [comment]
I think we'll go with human element. 
![default](https://user-images.githubusercontent.com/29916678/29356920-7a85ffe8-827e-11e7-8367-6b1744433929.jpg)
![default](https://user-images.githubusercontent.com/29916678/29356921-7be7c768-827e-11e7-851f-571e3d4c5905.jpg)

**Comment by Nur-alfayez** [comment]
@EmanHylooz

**Comment by EmanHylooz** [comment]
@Nur-alfayez both are nice by the way... we can have them both and A/B test. If there is no capacity for that, the human element looks more attractive. 

We still have a space to say something about the book?

**Comment by Nur-alfayez** [comment]
La of course we have capacity. Will do both.

Didnt check space thing yet will keep you posted or delegate to Farah.

**Comment by EmanHylooz** [comment]
the book الخبز الحافي  is very very dramatic, and full of blood and sex lol and محمد شكري is very sad person /// he learned how to write at the age 20 ! he was completely ignorant and poor before ... 

this picture really shows how he is"
https://www.google.jo/search?q=%D9%85%D8%AD%D9%85%D8%AF+%D8%B4%D9%83%D8%B1%D9%8A&rlz=1C1CHBF_enUS715JO716&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiz-Iy7udvVAhVCvBQKHXztDKEQ_AUICigB&biw=1216&bih=632&dpr=2.25#imgrc=-nxy9oe5W4BVsM:

or other pictures in google images that will work better in terms of resolution and has the same sadness.

**Comment by Nur-alfayez** [comment]
Sure will use this one, any recommendations about the other books images?

**Comment by Nur-alfayez** [comment]
@Farahalfayez

**Comment by Farahalfayez** [comment]
Hi guys, we can test both authors cards and titles cards against each other in 2 campaigns. This would increase the budget to around $100 (maybe less depending on performance and frequency). Are you ok with that? @EmanHylooz @Nur-alfayez

**Comment by EmanHylooz** [comment]
@Farahalfayez @Nur-alfayez Yes sounds reasonable. I am thinking the test results can be used to do the same campaign (the one that got more clicks) to the rest of the users who has the app and did not start free trial yet, we have a good number of those. ya3ny ne3temedha as a style for re-targeting using the coming soon books  or other interesting book titles we already have.

**Comment by Nur-alfayez** [comment]
I'll keep this one open so we continue discussions on campaign results here. @Farahalfayez @EmanHylooz
